src/App/App.js

```diff
@@ -10,11 +9,19 @@ class App extends Component {
     super()
     this.state = {
       movieId: null,
-      movies: [...testData],
-      query: ""
+      movies: [],
+      query: "",
+      err: ""
     }
   }
 
+  componentDidMount() {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+      .then(response => response.json())
+      .then(({ movies }) => this.setState({ movies: movies}))
+      .catch(err => this.setState({ err: err }))
```

Comment:
I don't _think_ this will catch status code 4xx and 5xx errors, since those aren't technically network errors. Here's a [stackoverflow](https://stackoverflow.com/questions/38235715/fetch-reject-promise-and-catch-the-error-if-status-is-not-ok) I was checking out on this.

I think the `response.ok` property of the promise object is where we'll need to look to catch status code errors, so we'll need to add a `.then()` with a conditional to throw an Error object if `!response.ok`, since right now I think it'll look at a status code 4xx or 5xx response as a valid response and be happy that the promise has either been successfully returned or rejected without really caring that the response was an error.